python/cudf/cudf/testing/_utils.py

```diff
@@ -306,7 +306,11 @@ def gen_rand(dtype, size, **kwargs):
             np.random.randint(low=low, high=high, size=size), unit=time_unit
         )
     elif dtype.kind in ("O", "U"):
-        return pd.util.testing.rands_array(10, size)
+        low = kwargs.get("low", 10)
+        high = kwargs.get("high", 11)
+        return pd.util.testing.rands_array(
```

Comment:
I know that you didn't add this, but since you're changing it we may as well fix it. `pandas.util.testing` has been deprecated for over 2 years now, so if the current implementation is causing issues for you we should probably rework it altogether. Perhaps try out some of the replacements proposed [here](https://stackoverflow.com/questions/40461072/generate-random-strings-in-pandas)? I don't have an immediate best solution for this, sorry. Let me know if you'd like more input.

Reply:
This is being handled along with the other deprecation warnings in `test_orc.py` in [10772](https://github.com/rapidsai/cudf/pull/10772). I will merge in this change since I didn't add the functionality of min/max to that PR.